tests/workflow_tests/test_events.py

```diff
@@ -92,6 +92,10 @@ def test_paginated_events(self):
         self.assertEquals(pagination_info.offset, offset)
         self.assertEquals(pagination_info.size, size)
 
+    def test_query_with_reserved_characters(self):
```

Comment:
maybe also add one more test which checks more than the request merely not crashing, but rather actually try to retrieve a real event which contains one of these characters?